packages/react-core/src/components/Progress/ProgressContainer.tsx

```diff
@@ -3,6 +3,7 @@ import progressStyle from '@patternfly/react-styles/css/components/Progress/prog
 import { css } from '@patternfly/react-styles';
 import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
 import TimesCircleIcon from '@patternfly/react-icons/dist/js/icons/times-circle-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
```

Comment:
Change import path to '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';`

Reply:
Good catch! thanks

Reply:
Oh the snapshot needs update as well. Will repost in 1h